### PR TITLE
Updated considerations and limitations

### PR DESCRIPTION
**Pull request by saurabhr14**
Updated considerations and limitations for Sovereign Clouds

### CODE REVIEW

File: docs/admin/feature-usage-adoption.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ This section lists the report's considerations and limitations.
 
 * During business continuity and disaster recovery, the *Audit* table may have data quality issues.
 
+* Audit log retention period for the Feature Usage and Adoption dataset/report is currently rolling 30 days only.
```
Comment (KesemSharabi) [inline docs/admin/feature-usage-adoption.md:103]: Please change to:

The report retains information for 30 days.

File: docs/admin/monitoring-workspace.md [hunk 2]
```diff
@@ -50,6 +50,10 @@ There are several ways you can manage access to content of the admin monitoring
 
 * **Dataset** - You can [share access to a dataset](/power-bi/connect-data/service-datasets-share) with other users. Once a dataset is shared, you can't unshare it.
 
+## Considerations and limitations
+
+* Sovereign Clouds (GCC) are not yet supported in the initial public preview release.
```
Comment (KesemSharabi) [inline docs/admin/monitoring-workspace.md:55]: Is this for all sovereign clouds or only CGS? I assume it's for all clouds so please change to this:

Sovereign clouds are not supported.

**Review by KesemSharabi — CHANGES_REQUESTED** [review]

**Comment by KesemSharabi** [comment]
#sign-off

**Comment by saurabhr14** [comment]
#sign-off

**Comment by chi-qin** [comment]
#sign-off

**Comment by chi-qin** [comment]
#sign-off

**Comment by chi-qin** [comment]
#sign-off

**Comment by KesemSharabi** [comment]
@chi-qin - there's a bug that's not letting us sign off.

**Comment by chi-qin** [comment]
@KesemSharabi Yeah, I am the incident owner, I just use this for testing. Sry for some noise

**Comment by KesemSharabi** [comment]
@chi-qin, no worries and thanks for trying to fix this!

**Pull request merged**